tools/build_and_install.py

```
@@ -286,11 +340,161 @@ def build_go_agent(
+        print(f"  {Console.warn(t('warning'))} {t('vswhere_not_found')}")
+        return False
+
+    # 用 vcvarsall.bat 按目标架构初始化（x86_64->x64，aarch64->arm64）
+    vcvarsall = Path(vs_path) / "VC" / "Auxiliary" / "Build" / "vcvarsall.bat"
+    if not vcvarsall.exists():
+        print(
+            f"  {Console.warn(t('warning'))} {t('vcvars_not_found', path=vcvarsall)}"
+        )
+        return False
+
+    # vcvarsall 参数：x86_64 -> x64（x64 host -> x64 target）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)